tests/runtests.pl

```diff
@@ -5342,6 +5347,13 @@ sub disabledtests {
     }
     $TESTCASES = $verified;
 }
+if($repeat) {
+    my $s;
+    for(1 .. $repeat) {
+        $s .= $TESTCASES;
```

Comment:
shouldn't a space be present like "$TESTCASES "

I've been doing it like this:
~~~
./runtests.pl -p `printf '230 %.0s' {1..100}`
~~~

Reply:
`$TESTCASES` already have the necessary spaces so this should work fine! I didn't try with all but I've verified it with a limited set of tests (including with `-R`).

> I've been doing it like this:

I've done similar things too but I always forget the exact syntax so I figured it could be fun to add "native" support for it!